spec/acceptance/backup_spec.rb

```diff
@@ -8,7 +8,7 @@ class { 'pe_databases':
        }#{' '}
        MANIFEST
 
-    # Run it twice and test for idempotency
-    idempotent_apply(pp)
+    # Expect a change due to backup notify
+    apply_manifest(pp, expect_changes: true)
```

Comment:
Do you mind adding this line twice? That way we still test running it multiple times doesn't throw errors.

```suggestion
apply_manifest(pp)
apply_manifest(pp, expect_changes: true)
```